example/index.html

```diff
@@ -6,6 +6,7 @@
   <style>
     body { 
       font-family: Helvetica;
+      background: linear-gradient(90deg, #69b7eb, #b3dbd3, #f4d6db);
```

Comment:
atm we can't add custom styles to the static page, because of how ts-react-toolbox works (we can maybe improve that). Instead, we have to use 
```javascript
import {injectGlobal} from 'styled-components'

injectGlobal`
  background: linear-gradient(90deg, #69b7eb, #b3dbd3, #f4d6db);
`
```